pimd/pim_cmd.c

```diff
@@ -7331,12 +7331,12 @@ DEFUN (ip_pim_keep_alive,
 
 DEFUN (no_ip_pim_keep_alive,
        no_ip_pim_keep_alive_cmd,
-       "no ip pim keep-alive-timer (31-60000)",
+       "no ip pim keep-alive-timer [(1-65535)]",
        NO_STR
        IP_STR
        "pim multicast routing\n"
        "Keep alive Timer\n"
-       "Seconds\n")
+       IGNORED_IN_NO_STR)
 {
 	const char *vrfname;
 	char ka_timer[5];
```

Comment:
Same here: either stringnify define or use `DESTROY` in the YANG op.

Reply:
change to destroy